src/azure-cli/requirements.py3.Darwin.txt

```diff
@@ -124,7 +124,7 @@ PyNaCl==1.5.0
 pyOpenSSL==24.0.0
 python-dateutil==2.8.0
 requests-oauthlib==1.2.0
-requests[socks]==2.31.0
+requests[socks]==2.32.0
```

Comment:
We use `requests[socks]` instead of `requests`, so dependabot can't create PR automatically.

Reply:
Any specific reason we don't want PRs created automatically for requests? Or should I change it to just `requests`?